file-system.ts

```diff
@@ -132,7 +133,8 @@ export class FileSystem implements IFileSystem {
 
 		return future;
 	}
-
+	
+	@decorators.promisify("fs")
```

Comment:
Is the parameter `"fs"` really needed here? Can't it be resolved from the class level decorator?